exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -211,38 +216,41 @@ void setInvalidValidatorCount() {
     Class<IllegalArgumentException> exceptionType = IllegalArgumentException.class;
     short invalidValidatorCount = 0;
     TestKit.Builder testKitBuilder = TestKit.builder()
-        .withService(TestServiceModule.class);
+        .withDeployedService(ARTIFACT_ID, ARTIFACT_FILENAME)
+        .withService(ARTIFACT_ID, SERVICE_NAME, SERVICE_ID,
+            Any.getDefaultInstance());
     assertThrows(exceptionType, () -> testKitBuilder.withValidators(invalidValidatorCount));
   }
 
   @Test
   void requestWrongServiceClass() {
     Class<IllegalArgumentException> exceptionType = IllegalArgumentException.class;
     try (TestKit testKit = TestKit.builder()
-        .withService(TestServiceModule.class)
+        .withDeployedService(ARTIFACT_ID, ARTIFACT_FILENAME)
+        .withService(ARTIFACT_ID, SERVICE_NAME, SERVICE_ID,
+            Any.getDefaultInstance())
         .build()) {
       assertThrows(exceptionType,
-          () -> testKit.getService(TestService.SERVICE_ID, TestService2.class));
+          () -> testKit.getService(SERVICE_NAME, TestService2.class));
     }
   }
 
   @Test
-  void requestWrongServiceId() {
+  void requestWrongServiceId(TestKit testKit) {
     Class<IllegalArgumentException> exceptionType = IllegalArgumentException.class;
-    try (TestKit testKit = TestKit.forService(TestServiceModule.class)) {
-      assertThrows(exceptionType, () -> testKit.getService((short) -1, TestService2.class));
-    }
+    assertThrows(exceptionType, () -> testKit.getService("Invalid service name",
+        TestService.class));
   }
 
   @Test
   void createTestKitMoreThanMaxServiceNumber() {
     Class<IllegalArgumentException> exceptionType = IllegalArgumentException.class;
-    List<Class<? extends ServiceModule>> serviceModules = new ArrayList<>();
+    TestKit.Builder testKitBuilder = TestKit.builder();
     for (int i = 0; i < TestKit.MAX_SERVICE_NUMBER + 1; i++) {
-      serviceModules.add(TestServiceModule.class);
+      testKitBuilder = testKitBuilder.withDeployedService(ARTIFACT_ID, ARTIFACT_FILENAME)
```

Comment:
(1) Why is the same artifact deployed N times if that is not what is tested?
(2) Why the same instance (name and id) is attempted to be added?